.eslintrc.json

```diff
@@ -7,6 +7,7 @@
   },
   "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
   "rules": {
+    "comma-dangle": ["error", "always-multiline"],
```

Comment:
do we need eslint rule for this? we are using prettier

Reply:
the lint rule gives you in-editor warnings when you're writing code (helps in case you don't have format on save turned on).
will also give a better warning in the CI logs. (the prettier check will just say the file failed formatting, whereas the eslint check will tell you the actual problem)